installation/proc-install-rhdh-ocp-operator.adoc

```diff
@@ -1,28 +1,37 @@
-[id='proc-install-rhdh-ocp-operator_{context}']
-
+[id="proc-install-rhdh-ocp-operator_{context}"]
 = Deploying {product} on {ocp-short} using the Operator
 
-As a developer, you can install {product} on your OpenShift Container Platform using the {product} Operator.
+As a developer, you can deploy a {product} instance on {ocp-short} by using the *Developer* perspective in the web console.
 
 .Prerequisites
-* Your administrator has installed the {product} Operator. For more information, see xref:proc-install-operator_admin-rhdh[Installing the {product} Operator].
-* You have created a project in {ocp-short} for your {product} instance. For more information about creating a project in OpenShift, see link:https://docs.openshift.com/container-platform/4.14/applications/projects/working-with-projects.html#odc-creating-projects-using-developer-perspective_projects[Red Hat OpenShift documentation].
+
+* A cluster administrator has installed the {product} Operator. For more information, see xref:proc-install-operator_admin-rhdh[Installing the {product} Operator].
 
 .Procedure
-. From the *Developer* perspective on the {product-short} web console, click *+Add*.
+
+. Create a project in {ocp-short} for your {product} instance, or select an existing project.
++
+[TIP]
+====
+For more information about creating a project in {ocp-short}, see link:https://docs.openshift.com/container-platform/4.15/applications/projects/working-with-projects.html#creating-a-project-using-the-web-console_projects[Creating a project by using the web console] in the {ocp-brand-name} documentation.
+====
+. From the *Developer* perspective on the {ocp-short} web console, click *+Add*.
 . From the *Developer Catalog* panel, click *Operator Backed*.
 . In the *Filter by keyword* box, enter _{product-short}_ and click the *{product}* card.
 . Click *Create*.
-. Optionally, configure the {product-short} instance with non-default settings.
-. From the *{product}* page, click *Create*.
-. From the *Create Backstage* page, click *Create*
-. From the *Topology* tab, click the *Open URL* icon on the {product-short} pod to start using the {product} platform.
+. Optionally, configure the {product-short} instance with custom settings.
```

Comment:
To be consistent with the customization method that we promote later, we might want to omit this step.

Reply:
True, and unlike the helm install, configuration CAN be done later without negative impact (AFAIK). 

With Helm if you don't set the correct cluster router base URL, you won't be able to open the RHDH application.

Reply:
Personally, I think that it would make more sense to document how to configure this, in full, here.
If I was a user reading our docs / installing the Operator, I would not assume that I would need to go and look at other documentation somewhere else later to finish configuring my setup. I would want to assume that everything I need to configure the Operator will be in one place.

I also think it's worth asking - it might be fine to change the config later without impact, but why should users expect a different workflow for Helm charts vs Operator installs? What happens if they create a Backstage instance without completing the additional configuration steps, and then configure it - are those instances updated to the new config, or are they negatively affected?

I would like to leave this as is for now, and discuss it with the team, since I think there is some work ongoing to create customer journeys for the installation procedure anyway atm. I think at the very least, we should indicate if we're taking this step out that you can finish this configuration later.

Reply:
can't we link from this section to the section on configuring the deployment? eg., for enabling dynamic plugins?

Reply:
A link to configuring the deployment might be nice. Also referring to `custom configuration` instead of `custom settings` will align with the remainder of the documentation.